utils/create-keying-sheet-tfm-facilities/mock-data/tfm-facility.ts

```diff
@@ -0,0 +1,36 @@
+import { Facility, TfmFacility } from '@ukef/dtfs2-common';
+import { ObjectId } from 'mongodb';
+
+/**
+ * Creates a tfm facility that can be used for utilisation reporting testing.
+ * It can be replaced by creating a deal with a facility manually and then pulling from the mongo db collection.
+ * This way we ensure we are inserting a valid facility.
+ * @param facilityId - id of the facility
+ * @param facilitySnapshot - a facility
+ * @param portalUserId - id of a portal user
+ * @returns a tfm facility
+ */
+export const aTfmFacility = (facilityId: ObjectId, facilitySnapshot: Facility, portalUserId: ObjectId): TfmFacility => ({
+  _id: facilityId,
+  facilitySnapshot,
+  tfm: {
+    hasBeenIssuedAndAcknowledged: true,
+    feeRecord: 5622.222222222223,
```

Comment:
should this number be not rounded?

Reply:
This facility was created by creating a facility on deployed env and then copying from mongo. If this should be being rounded then that would be a bug completely unrelated to this work.